contrib/IECoreMantra/src/IECoreMantra/procedural/Procedural.cpp

```diff
@@ -114,7 +114,7 @@ class VRAY_ieProcedural : public ProceduralPrimitive {
 	virtual void	 render();
 #if UT_MAJOR_VERSION_INT >= 16
 	UT_StringHolder m_className;
-	int64 m_classVersion;
+	int32 m_classVersion;
```

Comment:
Either this should remain int64 or it needs changing in `VRAY_ieProcedural::initialize()` as well.